src/stories/Toast.story.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <Story title="Toast">
+    <Toaster richColors />
+    <Variant title="Legacy Toast">
+      <button @click="launchLegacyToast">Simple Legacy toast</button>
+    </Variant>
+    <Variant title="New Toast">
+      <button @click="openBasicToast()">Basic Toast</button>
+      <button @click="openSuccessToast">Success Toast</button>
+      <button @click="openErrorToast">Error Toast</button>
+    </Variant>
+  </Story>
+</template>
+
+<script lang="ts" setup>
+import { Toaster } from "vue-sonner"
```

Comment:
Are we not having a wrapper around `Toaster`? How does the mechanism work if I were to use Toaster inside the main app since `vue-sonner` is installed only in the UI package?